srcpkgs/neovim/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'neovim'
 pkgname=neovim
-version=0.3.8
+version=0.4.2
 revision=1
 build_style=cmake
 configure_args="-DENABLE_JEMALLOC=0"
```

Comment:
Remove this line, jemalloc is not used anymore https://github.com/neovim/neovim/pull/9526.